utilities/merge_operators/sortlist.h

```diff
@@ -0,0 +1,39 @@
+//  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+// A test MergeOperator for RocksDB that implements Merge Sort.
+// It is built using the MergeOperator interface. This is useful for
+// testing/benchmarking
```

Comment:
Can you describe a little bit more in the comment about what the class actually does?

I'm picky but based on Google C++ Style, this seems to be a class comment rather than file comment and should be placed before the class declaration: https://google.github.io/styleguide/cppguide.html#File_Comments